doc/classes/CPUParticles.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CPUParticles" inherits="GeometryInstance" category="Core" version="3.2">
 	<brief_description>
+		CPU-based 2D particle emitter.
 	</brief_description>
 	<description>
+		CPU-based 3D particle node used to create a variety of particle systems and effects. [Particles2D] features an emitter that generates some number of particles at a given rate.
```

Comment:
Linking to [Particles2D], should be [Particles].

Maybe the description could mention that `Particles` provides hardware acceleration? As this node is mostly for compatibility purposes, I think it'd be useful to mention you mostly want to use it if you're targeting older devices.